qa/suites/rgw/bucket-logging/tasks/s3tests.yaml

```diff
@@ -0,0 +1,10 @@
+tasks:
+- install:
+- ceph:
+- rgw: [client.0]
+- tox: [client.0]
```

Comment:
> @cbodley i'm getting this error consistently in the test. can this be related to the change here?

these tasks are already run as part of `0-install.yaml`, which is why you see `install` trying to run these mkfs.xfs commands a second time

Reply:
http://qa-proxy.ceph.com/teuthology/yuvalif-2024-11-14_16:50:25-rgw:bucket-logging-wip-yuval-bucket-logging3-distro-default-smithi/7994195/teuthology.log shows the expanded list of tasks near the top:
```
tasks:
  - install: null
  - ceph: null
  - openssl_keys: null
  - rgw:
    - client.0
  - tox:
    - client.0
  - install: null
  - ceph: null
  - rgw:
    - client.0
  - tox:
    - client.0
  - s3tests:
      client.0:
        boto3_extensions: true
        extra_attrs:
        - bucket_logging
        rgw_server: client.0
```